src/plays/keeper/Keeper.jsx

```diff
@@ -0,0 +1,78 @@
+
+import { useLocation } from 'react-router-dom';
+import { getPlayById } from 'meta/play-meta-util';
+
+import PlayHeader from 'common/playlists/PlayHeader';
+import './Keeper.css'
+import Header from './Header';
+import CreateArea from './CreateArea'; 
+import Footer from './Footer';
+import { useState, useEffect } from 'react';
+import Note from './Notes';
+
+  //Get the Local Storage
+
+const getLocalItems = () => {
+  let lists = localStorage.getItem('list');
```

Comment:
It is a great opportunity to write a hook for handling localstorage. If you want to learn and do so, please keep the hook under common/hooks folder.

Reply:
I think it will be easy for beginners to understand hooks and they can easily relate to them if, it is written inside the same file. They can easily relate to the variables and understand them.